openedx/features/caliper_tracking/tests/tests.py

```diff
@@ -0,0 +1,50 @@
+"""
+
+"""
+
+import json
+import os
+from datetime import datetime
+
+from django.test import TestCase
+import logging
+
+from openedx.features.caliper_tracking.base_transformer import CaliperBaseTransformer
+from openedx.features.caliper_tracking.caliper_config import EVENT_MAPPING
+
+
+TEST_DIR_PATH = 'openedx/features/caliper_tracking/tests/'
+
+
+class CaliperTransformationTestCase(TestCase):
+    """Data driven test case that tests all transformers for expected output"""
+
+    maxDiff = None
```

Comment:
The reason I added it is that it is necessary to see the event name and structure for our events

because we are doing data driven tests, this single test case will cover all 200+ events
we can't know which event is failing without this being set